lib/download.test.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const { getAssetsToDownload } = require("./download");
+const assert = require("assert");
+
+const allAssets = [
+  { url: "https://bogus1.com", name: "win32-ia32-node.napi.glibc.node" },
+  { url: "https://bogus2.com", name: "win32-x64-node.napi.glibc.node" },
+  { url: "https://bogus3.com", name: "linux-x64-node.napi.glibc.node" },
+  { url: "https://bogus4.com", name: "linux-arm64-node.napi.glibc.node" },
+  { url: "https://bogus5.com", name: "linux-arm-node.napi.glibc.node" },
+  { url: "https://bogus6.com", name: "linux-x64-musl-node.napi.musl.node" },
+  { url: "https://bogus7.com", name: "darwin-x64.arm64-node.napi.glibc.node" },
+];
+
+const test = (name, fn) => {
+  fn();
+  console.log(`\x1b[32m%s\x1b[0m`, `✓ `, name);
```

Comment:
Super light weight tests, without any packages
Pulling in mocha or jest or the like is just overkill for this (end up with a lot of CG nighmare)
Besides this is a very tiny npm package with small components, but would like to have some tests without any dependencies

Reply:
I'd say that mocha would be OK to depend on since dev-dependencies don't affect the size of the published npm package (IIRC). But this works too.

Reply:
Its not the size, but the management of component goveranance (using mocha or jest ends up pulling a tonne of depencenies and its just not worth the effor)